xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-docker/src/main/java/org/xwiki/test/docker/junit5/TestConfiguration.java

```diff
@@ -484,6 +492,22 @@ public void setVNC(boolean vnc)
         this.vnc = vnc;
     }
 
+    /**
+     * @return true if WCAG rules should be checked.
+     */
+    public boolean isWcagEnabled()
```

Comment:
Missing `@since` annotation

Reply:
Small preference to use WCAG and not `Wcag` since it's an acronym. So either `isWCAG()` or `isWCAGEnabled()`. See also `getExtraJARs()`, `getJDBCDriverVersion()`, etc.

Reply:
note that `isWCAG()` might be more consistent with the other `isVerbose()`, `isOffice()`, etc.